core/src/main/java/nl/nn/adapterframework/parameters/Parameter.java

```diff
@@ -540,13 +540,13 @@ private Object getValueAsType(Object message, boolean namespaceAware) throws Par
 		Message request = Message.asMessage(message);
 		Object result = message;
 		try {
+			Object requestObject = request.asObject();
```

Comment:
kan je niet beter message gebruiken?

Reply:
Dan moet ik vijf keer asObject doen

Reply:
nee ik bedoel niet Message, ik bedoel message (argument vd. methode)

Reply:
Object message wordt ook gebruikt als result, en die kan dat blijven. Het is dus niet goed, en heeft ook geen zin om de parameter Object message te veranderen in een Message.
Ik zal voor de verduidelijking wel de namen veranderen.